src/assets/_project/_blocks/utils/qg-load-google-api.js

```diff
@@ -19,7 +19,7 @@
     };
     if ($('#googleapi').length <= 0) {
       let s = document.createElement('script');
-      let u = 'https://maps.googleapis.com/maps/api/js?key=googleMapsApiKey&region=AU&libraries=places';
+      let u = `https://maps.googleapis.com/maps/api/js?key=${window.qg.googleKey}&region=AU&libraries=places`;
```

Comment:
If any projects are using Google API and don't have this variable, then their components could potentially break

Reply:
This variable is registered by SWE3 'qg-env' file